relationships/synthesis/APM-APPLICATION-to-INFRA-AWSSQSQUEUE.yml

```diff
@@ -0,0 +1,26 @@
+relationships:
+  - name: extServiceCallsInfraAwsSqsQueue
```

Comment:
The file is named APM but the name has `extService`.
I wondered what it is supposed to match and what's the correct naming ?

I've further seen that we are already receiving candidate relationships for APM. Wondered if that's known and if this definition still makes sense ?
(will share more details on internal channels)

Reply:
As we're already receiving candidate relationships for APM, I've updated the relationship files accordingly.